src/modules/pageIdentifier.js

```diff
@@ -0,0 +1,55 @@
+const SHOPGATE_PAGE_CART = 'cart';
+const SHOPGATE_PAGE_CHECKOUT_SUCCESS = 'checkout_success';
+const SHOPGATE_PAGE_CHECKOUT = 'checkout';
+const SHOPGATE_PAGE_LOGIN = 'login';
+const SHOPGATE_PAGE_REGISTER = 'register';
+const SHOPGATE_PAGE_REGISTER_SUCCESS = 'register_success';
+const SHOPGATE_PAGE_FORGOT_PASSWORD = 'forgot_password';
+
+export const pageIdentifier = {
```

Comment:
At the moment the code is just used in one place. I don't see the advantage here - the logic belongs to each other. If you don't want to leave it as it is, I will refactor this to be a class because the code here is basically a class.

Reply:
changed in order to reduce amount of code transpiled in the final Java Script